testsuite/documentation/cucumber-steps.md

```diff
@@ -615,7 +616,8 @@ The virtual machine is created without Uyuni, directly on the virtual host
 using `qemu-img` and `virt-install`
 
 ```gherkin
-  When I create "test-vm" virtual machine on "virt-server"
+  When I create a leap virtual machine named "test-vm" without cloudinit on "virt-server"
+  When I create a sles virtual machine named "test-vm" with cloudinit on "virt-server"
```

Comment:
And

Reply:
Those are just examples in the documentation. We have other occurrences where we do it the same way.